third-party/libRocket/Source/Core/Dictionary.cpp

```diff
@@ -479,10 +479,9 @@ void Dictionary::Clear()
 	DictionaryEntry *ep;
 	bool table_is_malloced;
 	int n_full;
-	int i, n;
+	int i;
```

Comment:
n is still used on Line 493 in an assert statement

Reply:
I love that it passes build on Linux, it appears that ROCKET_ASSERT is defined to empty